packages/react/src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -153,6 +155,8 @@ export default {
   add,
   'arrow-up': arrowUp,
   'arrow-down': arrowDown,
+  'arrow-left': arrowLeft,
```

Comment:
Let only design add icons. Use `arrow-up` and rotate it as needed.

Reply:
The two icons `arrow-left`, `arrow-right` are simply copied from `processedIcons` folder. I did not design them.